.github/workflows/ci.yml

```
@@ -0,0 +1,222 @@
+name: Rust CI/CD
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+env:
+  IMAGE: paritytech/ci-unified:bullseye-1.73.0
+
```

```suggestion
concurrency:
  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
  cancel-in-progress: true
```
Let's save some money ;)